recipe-server/client/control/components/Fields.js

```diff
@@ -23,13 +23,14 @@ export function buildControlField({
   label,
   className = '',
   InputComponent,
+  hideErrors = false,
```

Comment:
Technically the `= false` shouldn't be needed, but I've been a fan of setting defaults this way lately, so I think it's fine anyway.